openslides_backend/models/models.py

```diff
@@ -0,0 +1,1493 @@
+        related_name="referenced_in_motion_recommendation_extension_ids",
+    )
+    referenced_in_motion_recommendation_extension_ids = fields.RelationListField(
+        to=Collection("motion"),
+        related_name="recommendation_extension_reference_ids",
+        generic_relation=True,
+    )
+    category_id = fields.RelationField(
+        to=Collection("motion_category"), related_name="motion_ids"
+    )
+    block_id = fields.RelationField(
+        to=Collection("motion_block"), related_name="motion_ids"
+    )
+    submitter_ids = fields.RelationListField(
+        to=Collection("motion_submitter"), related_name="motion_id"
+    )
+    supporter_ids = fields.RelationListField(
+        to=Collection("user"),
+        related_name="supported_motion_$_ids",
+        structured_relation=["meeting_id"],
+    )
+    poll_ids = fields.RelationListField(
+        to=Collection("motion_poll"), related_name="motion_id"
+    )
+    change_recommendation_ids = fields.RelationListField(
+        to=Collection("motion_change_recommendation"), related_name="motion_id"
+    )
+    statute_paragraph_id = fields.RelationField(
+        to=Collection("motion_statute_paragraph"), related_name="motion_ids"
+    )
+    comment_ids = fields.RelationListField(
+        to=Collection("motion_comment"), related_name="motion_id"
+    )
+    agenda_item_id = fields.RelationField(
+        to=Collection("agenda_item"),
+        related_name="content_object_id",
+        generic_relation=True,
+    )
+    list_of_speakers_id = fields.RelationField(
+        to=Collection("list_of_speakers"),
+        related_name="content_object_id",
+        generic_relation=True,
+        required=True,
+    )
+    tag_ids = fields.RelationListField(
+        to=Collection("tag"), related_name="tagged_ids", generic_relation=True
+    )
+    attachment_ids = fields.RelationListField(
+        to=Collection("mediafile"), related_name="attachment_ids", generic_relation=True
+    )
+    projection_ids = fields.RelationListField(
+        to=Collection("projection"), related_name="element_id", generic_relation=True
+    )
+    current_projector_ids = fields.RelationListField(
+        to=Collection("projector"),
+        related_name="current_element_ids",
+        generic_relation=True,
+    )
+    personal_note_ids = fields.RelationListField(
+        to=Collection("personal_note"),
+        related_name="content_object_id",
+        generic_relation=True,
+    )
+    meeting_id = fields.RelationField(
+        to=Collection("meeting"), related_name="motion_ids", required=True
+    )
+
+
+class MotionSubmitter(Model):
+    collection = Collection("motion_submitter")
+    verbose_name = "motion submitter"
+
+    id = fields.IntegerField()
+    weight = fields.IntegerField(constraints={"default": 0})
+    user_id = fields.RelationField(
+        to=Collection("user"),
+        related_name="submitted_motion_$_ids",
+        structured_relation=["meeting_id"],
```

Comment:
This doesn't work since the `meeting_id` doesn't exist as a field

Reply:
We have to add e `through=[..]` keyword which is not implemented yet.

Reply:
Should be done now.